plugins/node/instrumentation-fs/src/instrumentation.ts

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as api from '@opentelemetry/api';
+import { isTracingSuppressed, suppressTracing } from '@opentelemetry/core';
+import {
+  InstrumentationBase,
+  InstrumentationNodeModuleDefinition,
+  isWrapped,
+} from '@opentelemetry/instrumentation';
+import { VERSION } from './version';
+import {
+  ASYNC_FUNCTIONS,
+  PROMISE_FUNCTIONS,
+  SYNC_FUNCTIONS,
+} from './constants';
+import type * as fs from 'fs';
+import type { FsInstrumentationConfig } from './types';
+
+type FS = typeof fs;
+
+const hasPromises = parseInt(process.versions.node.split('.')[0], 10) > 8;
+
+export default class FsInstrumentation extends InstrumentationBase<FS> {
+  constructor(protected override _config: FsInstrumentationConfig = {}) {
```

Comment:
It's preferable to not use use default value.
[See here](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/892#discussion_r805390997)

Reply:
:+1: